util/uuid.c

```diff
@@ -0,0 +1,32 @@
+#include "util/uuid.h"
+
+#include <uuid.h>
```

Comment:
This should be guarded behind `HAS_LIBUUID`.

Reply:
Oh, I guess both libuuid and FreeBSD's libc use `uuid.h`. Weird, please disregard this comment.